sdks/go/pkg/beam/io/bigqueryio/bigquery.go

```diff
@@ -208,29 +208,60 @@ func mustParseTable(table string) QualifiedTableName {
 // TODO(herohde) 7/14/2017: allow CreateDispositions and WriteDispositions. The default
 // is not quite what the Dataflow examples do.
 
+// WriteOptions represents additional options for executing a write
+type WriteOptions struct {
+	// CreateDisposition specifies the circumstances under which destination table will be created
+	CreateDisposition bigquery.TableCreateDisposition
+}
+
+// newWriteOptions creates a new instance of WriteOptions
+// "CreateIfNeeded" is set as the default write disposition
+func newWriteOptions() WriteOptions {
+	wo := WriteOptions{}
+	wo.CreateDisposition = bigquery.CreateIfNeeded
+	return wo
```

Comment:
Could be shortened into:

```suggestion
	return WriteOptions{CreateDisposition: bigquery.CreateIfNeeded}
```